movie-browser/src/view/FeaturedMovie.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import CardDetail from './CardDetail.js'
+
+// TODO: makes toggle for detail view work. have no idea why it's not working for this one
+// toggle: https://stackoverflow.com/questions/29913387/show-hide-components-in-reactjs
+class FeaturedMovie extends React.Component{
+  state = {isDetailShown: false}
```

Comment:
not a blocker but I would rather name this `shouldShowDetails` or `showDetails` or else the condition at line 23 doesn't seem right, semantic/english wise.

Reply:
Changed into "shouldShowDetails".
I had a fixed idea that status flag boolean should start with "is".

Reply:
it depends. for instance, `isNumberGreaterThanTen` or `isValidInput`. But for this should be `shouldShow` or `triggerDetails` or something like that.